psi4/src/psi4/libfunctional/LibXCfunctional.cc

```diff
@@ -214,22 +218,10 @@ double LibXCFunctional::query_density_cutoff() {
 void LibXCFunctional::set_omega(double omega) {
     omega_ = omega;
     user_omega_ = true;
-    if (xc_func_name_ == "XC_GGA_X_WPBEH") {
-        xc_gga_x_wpbeh_set_params(xc_functional_.get(), omega);
-    } else if (xc_func_name_ == "XC_GGA_X_HJS_PBE") {
-        xc_gga_x_hjs_set_params(xc_functional_.get(), omega);
-    } else if (xc_func_name_ == "XC_HYB_GGA_XC_LRC_WPBEH") {
-        xc_gga_x_wpbeh_set_params(xc_functional_->func_aux[0], omega);
-    } else if (xc_func_name_ == "XC_HYB_GGA_XC_WB97X") {
-        xc_functional_->cam_omega = omega;
-    } else if (xc_func_name_ == "XC_HYB_GGA_XC_WB97") {
-        xc_functional_->cam_omega = omega;
-    } else if (xc_func_name_ == "XC_HYB_GGA_XC_WB97X_V") {
-        xc_functional_->cam_omega = omega;
-    } else if (xc_func_name_ == "XC_HYB_GGA_XC_WB97X_D") {
-        xc_functional_->cam_omega = omega;
-    } else if (xc_func_name_ == "XC_HYB_MGGA_X_M11") {
-        xc_functional_->cam_omega = omega;
+    if ((xc_func_name_ == "XC_GGA_X_WPBEH") || (xc_func_name_ == "XC_GGA_X_HJS_PBE") || (xc_func_name_ == "XC_HYB_GGA_XC_LRC_WPBEH") ||
```

Comment:
All range-separated functionals in libxc should have `_omega` as a parameter.

The HSE functionals are an exception, since due to a bug the original version (now known as HSE03) uses different values of omega for DFT and for HF...